src/app/coursegrab/controllers/oauth2_callback_controller.py

```diff
@@ -0,0 +1,38 @@
+from os import environ
+import requests
```

Comment:
imports go first right?

Reply:
system imports before third party i think is the rule